enhancements/dns/externaldns-operator.md

```diff
@@ -155,22 +155,19 @@ metadata:
   namespace: my-externaldns-namespace
 spec:
   domains:
-    - my-cluster.my-domain.com
-  excludeDomains:
-    - apps.my-cluster.my-domain.com
+    - matchType: Exact
+      name: my-cluster.my-domain.com
+      filterType: Include
+    - matchType: Exact
+      name: apps.my-cluster.my-domain.com
+      filterType: Exclude
   provider:
     type: AWS
     credentials: my-secret-reference
-  sources:
-    - openshiftRoute:
-        ignoreHostnameAnnotation: "true"
-        labelSelector:
-          foo: bar
-    - crd:
-        resource: dnsrecords
-    - service:
-        namespace: foo
-        publishInternal: "true"
+  source:
+    type: OpenShiftRoute
+    hostnameAnnotationPolicy: Allow
+    AnnotationFilter: some.annotation.io/foo=bar
```

Comment:
Better to use a map:
```suggestion
    AnnotationFilter:
      "some.annotation.io/foo": bar
```
with this Go definition:
```go
    AnnotationFilter map[string]string `json:"annotationFilter,omitempty"`
```

Reply:
fixed.